requirements.txt

```diff
@@ -1,9 +1,10 @@
 numpy>=1.13.3
-pandas>=0.23.0
+pandas>=0.23.0,<0.24.0
 tqdm>=4.19.2
-featuretools>=0.7.0
+featuretools>=0.7.0,<0.12.0
```

Comment:
Same question with restricting featuretools

Reply:
reverted back to >=0.7.0